### PR TITLE
Update timeline

### PR DESCRIPTION
**Pull request by yijunx**
yijun's version of what to do in timeline

### CODE REVIEW

**Comment by yijunx** [comment]
wow

**Comment by erjunx** [comment]
this is some branch

File: timeline [hunk 1]
```diff
@@ -1,9 +1,16 @@
 Phase 1 (this might take several months)
 
-propose and valid (check market)
+propose and valid (check market), takes about 3 months
```
Comment (erjunx) [inline timeline:3]: is 3 month enough?

Comment (yijunx) [reply timeline:3]: kind of...

**Pull request merged**